rows_test.go

```diff
@@ -753,3 +755,60 @@ func ExampleRows_AddRows() {
 	// Output: scanned id: 1 and title: one
 	// scanned id: 2 and title: two
 }
+
+type MockStruct struct {
+	Type       int       `mock:"type"`
```

Comment:
Is this `mock:"type"` struct tag used anywhere? If not you may remove it.

Reply:
This is used to test reflect with Tag